crates/core/src/nats/conn_streams.rs

```diff
@@ -0,0 +1,70 @@
+use crate::nats::{
+    streams::Stream,
+    subjects::{BlockSubjects, TransactionSubjects},
+    NatsClient,
+    NatsError,
+};
+
+#[derive(Debug, Clone)]
+pub struct ConnStreams {
+    pub blocks: Stream<BlockSubjects>,
+    pub transactions: Stream<TransactionSubjects>,
+}
```

Comment:
Since Blocks & Transactions are variants of a stream, it would be better to express them as Enums

```suggestion
pub enum ConnStream {
    Blocks(Stream<BlockSubjects>),
    Transactions(Stream<TransactionSubjects>),
}
```

Reply:
This isn't possible because I need to keep the streams in some place in the memory since they are created at runtime, so if I added this here as a enum, I'll need another struct to save the streams, so kinda doesn't make sense